themes/default/content/blog/2022-03-30-introducing-pulumiverse/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Introducing the Pulumiverse"
+date: 2022-03-30T10:00:00Z
+draft: false
+meta_desc: The Pulumiverse is a community organized project to help people contribute and use Pulumi more effectively
+meta_image: meta.png
+authors: ["david-flanagan"]
+tags: ["Community"]
+---
+
+# Introducing the Pulumiverse
+
+Today, we're excited to announce that we're working with the Pulumi community to provide a place to interact and collaborate on Pulumi-based libraries, projects, and educational materials: the [Pulumiverse](https://github.com/pulumiverse).
+
+This project is especially important to me, because it's an idea that I've tried in the past, 2018 😯, and failed to commit to and give it the attention it deserves.
+
+Fortunately, two other wonderful Pulumi community members stepped up and started to put together their own programme and with me now being at Pulumi, the timing was amazingly serendipitous.
+
+I spoke with [Ringo De Smet](https://twitter.com/ringods), Head of CI/CD at Napoleon Games, and founding Pulumiverse member and he had this to say about why he's excited to be founding the Pulumiverse.
+
+> For infrastructure automation, we had other great tools which preceded Pulumi: Chef, Puppet, Ansible. Any of these tools have a vibrant community of people preventing that all of us need to re-invent the wheel. We had Sous-Chefs[1] for Chef & Vox Pupuli[2] for Puppet. But I see a lot of Pulumi users implementing the same stuff over and over again. That's why I think Pulumi needs it's own community. So I started it: Pulumiverse[3]
+
+[1] https://sous-chefs.org
+[2] https://voxpupuli.org
+[3] https://github.com/pulumiverse
```

Comment:
Side note: Without the preview, I can't check that these links render correctly in the quote. I'm pretty sure they need colons, though.

```suggestion
> For infrastructure automation, we had other great tools which preceded Pulumi: Chef, Puppet, Ansible. Any of these tools have a vibrant community of people preventing that all of us need to re-invent the wheel. We had Sous-Chefs[1] for Chef & Vox Pupuli[2] for Puppet. But I see a lot of Pulumi users implementing the same stuff over and over again. That's why I think Pulumi needs it's own community. So I started it: Pulumiverse[3]

[1]: https://sous-chefs.org
[2]: https://voxpupuli.org
[3]: https://github.com/pulumiverse
```